src/protocol.h

```diff
@@ -225,6 +225,17 @@ extern const char* GETBLOCKTXN;
  * @since protocol version 70014 as described by BIP 152
  */
 extern const char* BLOCKTXN;
+/**
+ * getcfheaders requests compact filter headers for a range of blocks.
+ * Only available with service bit NODE_COMPACT_FILTERS as described by
+ * BIP 157 & 158.
+ */
+extern const char *GETCFHEADERS;
+/**
+ * cfheaders is a response to a getcfheaders request containing a vector of
+ * filter headers for each block in the requested range.
```

Comment:
nit: technically, it's only one filter header plus a vector of filter hashes

Reply:
Thanks. I've corrected the comment.